shell/browser/electron_browser_main_parts.cc

```diff
@@ -439,8 +439,11 @@ void ElectronBrowserMainParts::PreMainMessageLoopRun() {
   content::WebUIControllerFactory::RegisterFactory(
       ElectronWebUIControllerFactory::GetInstance());
 
-  // --remote-debugging-port
   auto* command_line = base::CommandLine::ForCurrentProcess();
+  // --remote-debugging-pipe
+  if (command_line->HasSwitch(switches::kRemoteDebuggingPipe))
+    content::DevToolsAgentHost::StartRemoteDebuggingPipeHandler();
```

Comment:
I can see in Chromium they have their conditions set up so that `StartRemoteDebuggingPipeHandler` and `StartRemoteDebuggingServer` aren't called in the same instance.  Is it ok that we'd launch both the pipe and the server here if both arguments are passed in?

Reply:
Yes, it is totally fine. Chromium supports multiple concurrent CDP clients and they can be from both Web Sockets and Pipe. Content shell does [both](https://cs.chromium.org/chromium/src/content/shell/browser/shell_devtools_manager_delegate.cc?type=cs&q=StartRemoteDebuggingPipeHandler&sq=package:chromium&g=0&l=168). But since you asked, there is no reason to be smarter than Chrome here, so let me mimic the condition.

Reply:
Done!